containers/tefca-viewer/flyway/sql/V01_03__insert_hardcoded_values.sql

```diff
@@ -12,20 +12,20 @@ INSERT INTO valuesets VALUES('2_20240909','2','20240909','Cancer (Leukemia)','DI
 INSERT INTO valuesets VALUES('3_20240909','3','20240909','Cancer Leukemia','DIBBs','mrtc');
 
 -- Missing Syphilis valueset(s)
-INSERT INTO valuesets VALUES('4_20240909','4','20240909','Missing Syphilis mrtc','DIBBs','mrtc');
-INSERT INTO valuesets VALUES('5_20240909','5','20240909','Missing Syphilis dxtc','DIBBs','dxtc');
-INSERT INTO valuesets VALUES('6_20240909','6','20240909','Missing Syphilis sdtc','DIBBs','sdtc');
+INSERT INTO valuesets VALUES('4_20240909','4','20240909','Syphilis Medication','DIBBs','mrtc');
+INSERT INTO valuesets VALUES('5_20240909','5','20240909','Syphilis Diagnosis Problem','DIBBs','dxtc');
+INSERT INTO valuesets VALUES('6_20240909','6','20240909','Suspected Syphilis Disorder','DIBBs','sdtc');
 
 -- Missing Gonorhea valueset(s)
-INSERT INTO valuesets VALUES('7_20240909','7','20240909','Missing Gonorhea mrtc','DIBBs','mrtc');
-INSERT INTO valuesets VALUES('8_20240909','8','20240909','Missing Gonorhea dxtc','DIBBs','dxtc');
-INSERT INTO valuesets VALUES('9_20240909','9','20240909','Missing Gonorhea lotc','DIBBs','lotc');
-INSERT INTO valuesets VALUES('10_20240909','10','20240909','Missing Gonorhea lrtc','DIBBs','lrtc');
+INSERT INTO valuesets VALUES('7_20240909','7','20240909','Gonorhea Medication','DIBBs','mrtc');
+INSERT INTO valuesets VALUES('8_20240909','8','20240909','Gonorhea Diagnosis Problem','DIBBs','dxtc');
+INSERT INTO valuesets VALUES('9_20240909','9','20240909','Gonorhea Lab Organism','DIBBs','lotc');
+INSERT INTO valuesets VALUES('10_20240909','10','20240909','Gonorhea Lab Result','DIBBs','lrtc');
```

Comment:
```suggestion
INSERT INTO valuesets VALUES('7_20240909','7','20240909','Gonorrhea Medication','DIBBs','mrtc');
INSERT INTO valuesets VALUES('8_20240909','8','20240909','Gonorrhea Diagnosis Problem','DIBBs','dxtc');
INSERT INTO valuesets VALUES('9_20240909','9','20240909','Gonorrhea Lab Organism','DIBBs','lotc');
INSERT INTO valuesets VALUES('10_20240909','10','20240909','Gonorrhea Lab Result','DIBBs','lrtc');
```

this is my bad but I always misspell these